test/e2e/v1beta1/argo_workflow.py

```diff
@@ -63,6 +63,7 @@
     "trial-enas-cnn-cifar10-gpu":    "examples/v1beta1/trial-images/enas-cnn-cifar10/Dockerfile.gpu",
     "trial-enas-cnn-cifar10-cpu":    "examples/v1beta1/trial-images/enas-cnn-cifar10/Dockerfile.cpu",
     "trial-darts-cnn-cifar10":       "examples/v1beta1/trial-images/darts-cnn-cifar10/Dockerfile",
+    "trial-simple-pbt":              "examples/v1beta1/trial-images/simple-pbt/Dockerfile",
```

Comment:
Can you also add `suggestion-pbt` to `KATIB_IMAGES`?
Thanks for your contribution.

Reply:
Sorry about that messy commit; added the `suggestion-pbt`!